lib/timeline/component/ItemSet.js

```diff
@@ -258,6 +258,49 @@ ItemSet.prototype._create = function(){
   this.body.dom.centerContainer.addEventListener('mousemove', this._onMouseMove.bind(this));
   // right-click on timeline 
   this.body.dom.centerContainer.addEventListener('contextmenu', this._onDragEnd.bind(this));
+  this.body.dom.centerContainer.addEventListener('mouseout', this._onMouseOut.bind(this));
+
+  this.dragEvent = null;
+
+  this.touchParams.mouseOnLeft = this.touchParams.mouseOnRight= this.touchParams.mouseIsLeaving = false; 
+
+  this.setMouseOnLeft = function(isOn){
+    this.touchParams.mouseOnLeft = isOn;
+    this.setMouseLeave(false);
+  }
+
+  this.setMouseOnRight = function(isOn){
+    this.touchParams.mouseOnRight = isOn;
+    this.setMouseLeave(false);
+  }
+
+  this.setMouseLeave = function(isLeaveNow){
+    this.touchParams.mouseIsLeaving = isLeaveNow;
+
+    if((this.touchParams.mouseOnLeft || this.touchParams.mouseOnRight) && isLeaveNow) {
+      if(this.dragEvent) ItemSet.prototype._onDragEnd.bind(this)(this.dragEvent);
+      this.touchParams.mouseOnLeft = this.touchParams.mouseOnRight = false;
+    }
+
+  }
+
+  var context = this;
```

Comment:
the context is needed to use setMouseOnLeft function in the mouse events for example:       this.body.dom.panningZoneLeft.onmouseenter =  function(){
    context.setMouseOnLeft(true);
}